docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -30,114 +30,163 @@ See the WebLogic Server samples, [Domain home on a PV]({{< relref "/samples/simp
+  * `secrets`: A list of names of the Secrets for WebLogic [configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}) or model. If this field is specified, then the value of `spec.configOverrideSecrets` is ignored.
+  * `introspectorJobActiveDeadlineSeconds`: The introspector job timeout value in seconds. If this field is specified, then the operator's ConfigMap `data.introspectorJobActiveDeadlineSeconds` value is ignored. Defaults to 120 seconds.
+  
 * These elements are under `configuration.model`, only apply if the `domainHomeSourceType` is `FromModel`, and are discussed in [Model in Image]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}).
 
-  * `configMap`: Optional configuration map for supplying [runtime model file updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}) to Model in Image model configuration.
-  * `domainType`: Must be one of `WLS`, `JRF`, or `RestrictedJRF`.  Default is `WLS`.
-  * `runtimeEncryptionSecret`: Required. Expected field is `password`. This is used by Model in Image to encrypt data while the data is passed from the introspector job to WebLogic pods. The password can be arbitrary: the only requirement is that it must stay the same for the life of a domain resource. If a domain resource is deleted then redeployed, it's fine to change the password during the interim.
-  * `wdtEncryptionSecret`: Optional. Rarely used. See [Model in Image]({{< relref "/userguide/managing-domains/model-in-image/_index.md" >}}) for details.
+  * `configMap`: Name of a ConfigMap containing the WebLogic Deploy Tooling model.
+  * `domainType`: WebLogic Deploy Tooling domain type. Legal values: WLS, RestrictedJRF, JRF. Defaults to WLS.
+  * `runtimeEncryptionSecret`: Runtime encryption secret. Required when `domainHomeSourceType` is set to FromModel.
 
 * These elements are under `configuration.opss`, and only apply if the `domainHomeSourceType` is `FromModel` and the `domainType` is `JRF`.
 
-  * `walletPasswordSecret`: The expected secret field is `walletPassword`. Used to encrypt/decrypt the wallet that's used for accessing the domain's entries in its RCU database.
-  * `walletFileSecret`: Optional. The expected secret field is `walletFile`. Use this to allow a JRF domain to reuse its entries in the RCU database (specify a wallet file that was obtained from the domain home while the domain was booted for the first time).
+  * `walletPasswordSecret`: Name of a Secret containing the OPSS key passphrase.
+  * `walletFileSecret`: Name of a Secret containing the OPSS key wallet file.
```

Comment:
IMO, the descriptions of the wallet password/file secret fields have been stripped down too much.  See original wording.